src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -74,14 +74,16 @@
 
     * :borrow-timeout - The timeout when borrowing instances from the JRuby Pool
```

Comment:
Would be good to add a description of the `http-client-connect-timeout` and `http-client-socket-timeout` keys here for consistency.